lib/fittings/error_handler.js

```diff
@@ -6,6 +6,8 @@ var http = require('http');
 module.exports = function create() {
     var injector = require('../../index.js').injector;
     var swaggerService = injector.get('Http.Services.Swagger');
+    var nconf = injector.get('nconf');
```

Comment:
You should the `Services.Configuration` dependency here instead of `nconf` directly.